docs/Operators.md

```diff
@@ -15669,15 +15666,15 @@ x_original = length_resized > 1 ? start_x * (length_original - 1) + x_resized *
 <dd>Four modes: round_prefer_floor (default, as known as round half down), round_prefer_ceil (as known as round half up), floor, ceil. Only used by nearest interpolation. It indicates how to get "nearest" pixel in input tensor from x_original, so this attribute is valid only if "mode" is "nearest".</dd>
 </dl>
 
-#### Inputs (3 - 4)
+#### Inputs (2 - 4)
```

Comment:
Should this be 2 to 3? Because Sizes and Scales should not be specified together right.

Reply:
@askhade @hariharans29 I agree that here should be 2-3, but this line is generated automatically. Is there any way to overwrite it?